### PR TITLE
Update amazon.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

File: amazon.txt [hunk 1]
```diff
@@ -431,6 +431,16 @@ alb-aws-fr-bswx-2-1673521430.eu-central-1.elb.amazonaws.com
...
 alexa.amazon.com
 alexa.com
+mas-ssr.amazon.com
+mas-ssr-eu.amazon.com
+aviary.amazon.co.uk
+aviary.amazon.com
+aviary.amazon.de
+msh.amazon
+aviary.amazon
+device-metrics
```
Comment (nickspaargaren) [inline amazon.txt:441]:
```suggestion
```

This is not a valid domain, can you remove this one?

File: amazon.txt [hunk 2]
```diff
@@ -431,6 +431,16 @@ alb-aws-fr-bswx-2-1673521430.eu-central-1.elb.amazonaws.com
 alexa-sitestats.s3.amazonaws.com
 alexa.amazon.com
 alexa.com
+mas-ssr.amazon.com
+mas-ssr-eu.amazon.com
+aviary.amazon.co.uk
+aviary.amazon.com
+aviary.amazon.de
+msh.amazon
```
Comment (nickspaargaren) [inline amazon.txt:439]:
```suggestion
```

https://whois.domaintools.com/msh.amazon

File: amazon.txt (hunk 1 above)
Comment (gallegonovato) [reply amazon.txt]: Done :)

File: amazon.txt (hunk 2 above)
Comment (gallegonovato) [reply amazon.txt]: Done :)

File: amazon.txt [hunk 3]
```diff
@@ -778,6 +785,11 @@ delivery-cdn-cf.adswizz.com
 delivery.vidible.tv
 det-ta-g7g.amazon.com
 dev-to.s3.us-east-2.amazonaws.com
+ec2-63-35-76-112.eu-west-1.compute.amazonaws.com
+ec2-54-72-248-64.eu-west-1.compute.amazonaws.com
+ec2-34-241-84-150.eu-west-1.compute.amazonaws.com
+ec2-63-32-155-164.eu-west-1.compute.amazonaws.com
+a92-122-104-90.deploy.static.akamaitechnologies.com
```
Comment (nickspaargaren) [inline amazon.txt:792]:
```suggestion
```

This is not an Amazon domain: https://whois.domaintools.com/akamaitechnologies.com. Please check your domains if you're not sure if it's Amazon.

Comment (gallegonovato) [reply amazon.txt:792]: Done

**Pull request merged**